webapp/store/app/error/state.js

```diff
@@ -0,0 +1,9 @@
+export const appStateKey = '_app'
+export const stateKey = 'error'
+
+const isEmpty = (val) => val === undefined || val == null || val.length <= 0 || Object.values(val).length <= 0
+
+export const getSystemError = (state) =>
+  state && state[appStateKey] && state[appStateKey][stateKey] && !isEmpty(state[appStateKey][stateKey])
```

Comment:
maybe we can use R.path here and assign state[appStateKey][stateKey] to a variable